### ISSUE TITLE
Presentation slides

### ISSUE THREAD

**Issue by PelzKo** [PelzKo/immunedeconv2#43]
Here are the presentation slides.
Feel free to read through and write comments.

**Comment by PelzKo** [comment]
[immundedeconv2.pptx](https://github.com/PelzKo/immunedeconv2/files/5943492/immundedeconv2.pptx)

**Comment by grst** [comment]
Most of the points I already mentioned during the meeting: 

 * add slide numbers
 * reduce text on slides
 * avoid text-only slides, add some nice figures/illustrations
 
### Motivation
 * Start with the biological motivation, then explain deconvolution. 

### Methods
 * Only very brief overview, the talk is about your project, not the methods, after all

### Outlook
 * merge limitations and outlook 

Looking forward to the talk on Wednesday :)

**Comment by FFinotello** [comment]
Nice work!

Here are a few suggestions on the **Background/Motivation** slides:

1. In general, I would speak about "cell-type deconvolution" rather than "immune deconvolution", as the idea of second-generation deconvolution is to ultimately quantify *any* cell type.
2. Slide 4: I agree with @grst: biological intro first (what is the problem with bulk RNA-seq from heterogeneous samples?), then deconvolution (i.e., the solution).
3. Slide 6: I would not present _immunedeconv2_ yet, but just saying where you were aiming at: having an interface to second-generation deconvolution tools. _Immunedeconv2_ will be the protagonist of your next slides ;)
2. Slides 7-13: I agree with @grst that you could just summarize the major features of the methods included; Slide 13 is great in this sense, and you could add other relevant info to the table if needed. This slide might be presented later to describe which methods you have selected for implementation into the package.
3. Slide 13: how about "Strengths" instead of "Specialties"? CIBERSORTx is missing here.

As for the **format** to present your **Results** after the "Motivation" slide, I would reorganize the content starting from a summary of your contribution (R package, GitHub repo with continuous integration, documentation as manual + vignette, datasets...) followed by some slides explain each contribution in detail:
- R package (functions and options, implemented methods, parameter settings, input/output formats); example code might be helpful if short.
- Data import/conversion (code?)
- Embedded datasets (code for accessing them?)
- Manual/vignette (with figures!)
- ... other contributions?
- GitHub repo and continuous integration (with figures)
This can be followed by a test case: your (small) beanchmarking study. Here you can add plenty of figures (the scatterplots are definitely the most useful ones).

I would then add the 1-2 **Conclusions** slides to:
- Summarize your contribution and the interesting features of your package (Does it make the usage of the methods easier? Does it allow to exchange signatures between methods?...)
- Comment on the results of the small benchmarking
- Discuss current limitations and future work / desired features

And, one future "TO DO" might be to decide a nice name now that we can extend beyond immune cell deconvolution... any ideas? ;)

**Comment by yuyuleung** [comment]
[immundedeconv2.pptx](https://github.com/PelzKo/immunedeconv2/files/5946028/immundedeconv2.pptx)
I have reduced text in the first part. :)

**Comment by yuyuleung** [comment]
[immundedeconv2.pptx](https://github.com/PelzKo/immunedeconv2/files/5946239/immundedeconv2.pptx)
I have edited the background and motivation parts. :) When there is any problem, please let me know. Thank you!

**Comment by nicolasgoedert97** [comment]
[immundedeconv2.pptx](https://github.com/PelzKo/immunedeconv2/files/5950090/immundedeconv2.pptx)
Changed Second Generation Tools section and added Cibersort

Note: In the Background section text is overlapping, at least on my PC it looks like it.

**Comment by kathireinisch** [comment]
[immundedeconv2.pptx](https://github.com/PelzKo/immunedeconv2/files/5950820/immundedeconv2.pptx)
New benchmarking plots coming soon!
Background section text was overlapping on my laptop, too

**Comment by PelzKo** [comment]
Here is the updated version. Feel free to read through, correct errors or write comments
[immundedeconv2.pptx](https://github.com/PelzKo/immunedeconv2/files/5951013/immundedeconv2.pptx)

**Comment by grst** [comment]
Can't comment on the layout (on libre office on linux everything gets distorted) but content-wise it looks good to me now! 
Looking forward to the presentation. 

Btw, @FFinotello and I didn't get a zoom link yet for tomorrow. If you've got it, would you mind forwarding it to us?

**Comment by kathireinisch** [comment]
I also added the benchmarking scatterplot for the big dataset in the slides.
@grst You said something about the deconv results and the mRNA sizes yesterday, but I'm not sure if I still remember it correctly. What was the problem with the T cells again? :)
And regarding the link: We didn't get a zoom link either, but I assume it's the one from the lecture. That was the meeting where we split into groups and had our first team meeting in december. The links are personalized so i don't think i can give you mine, but you maybe still have your own?
[immundedeconv2.pptx](https://github.com/PelzKo/immunedeconv2/files/5953041/immundedeconv2.pptx)

**Comment by grst** [comment]
I see, I think I found mine! 

If a cell-type is off the diagonal that could be due to the different cell sizes (larger cell = more mRNA). With single cell TPM values, the read counts have been normalized per cell s.t. this information is lost. This would be one reason why it would be interesting to compare counts vs tpm.

**Comment by kathireinisch** [comment]
Ah yes, thanks! I added the prediction/ground thruth comparison to the vignette as well (at least for the faster methods, i'm going to add the other two tomorrow), but the t cell results actually didn't "improve".

Here are the two plots again: (unnormalized the one with 3 methods, tpm normalized with 5)
![scatter_unnormalized](https://user-images.githubusercontent.com/20580137/107427954-407e2100-6b22-11eb-84a6-eb24a15225b2.png)
![predictionVsGroundtruth](https://user-images.githubusercontent.com/20580137/107427965-4542d500-6b22-11eb-9ad0-9aab89efc155.png)

**Comment by grst** [comment]
Actually, the results are suprisingly similar!
Thanks, that's very interesting!

**Comment by FFinotello** [comment]
Indeed, very interesting. If you have several figures (e.g. more datasets or counts/TPM data types), you could show them in some "backup slides" at the end of your presentation. Looking forward to it!

**Comment by kathireinisch** [comment]
Yes, i'm adding them right now :) i'm also making some scatterplots with interchanged signature matrices, maybe we have time to show them, too!

**Comment by kathireinisch** [comment]
[immundedeconv2.pptx](https://github.com/PelzKo/immunedeconv2/files/5956828/immundedeconv2.pptx)
here is the latest version of the slides :)
